source/aks/patching-artifactory.html.md.erb

```diff
@@ -1,152 +1,111 @@
 ---
 title: Artifactory Patching
 weight: 30
-last_reviewed_on: 2024-03-13
+last_reviewed_on: 2024-06-06
 review_in: 12 months
 ---
 
 # <%= current_page.data.title %>
 
-Artifactory is utilized in the context of managing and updating software dependencies and configurations within a Kubernetes environment. Here's how it is used based on the provided instructions:
+Artifactory is utilized in the context of managing and updating software dependencies and configurations within a Kubernetes environment.
+Here's how it is used based on the provided instructions:
 
-The steps below will show you how to check the current version of Artifactory being used (7.71.5) and then to update it to a newer version (7.77.3). This process involves accessing Artifactory's website to determine the latest version available.
+The steps below will show you how to check the current **self-hosted version** of Artifactory being used and then to update it to a newer **self-hosted version**. This process involves accessing <[Artifactory's website](https://jfrog.com/help/r/jfrog-release-information/artifactory-release-notes)> to determine the latest **self-hosted version** version available.
```

Comment:
```suggestion
The steps below will show you how to check the current **self-hosted version** of Artifactory being used and then to update it to a newer **self-hosted version**. This process involves accessing [Artifactory's website](https://jfrog.com/help/r/jfrog-release-information/artifactory-release-notes) to determine the latest **self-hosted version** version available.
```

Could you remove the rest of these too please? They show up like this when rendered 
<img width="415" alt="image" src="https://github.com/hmcts/ops-runbooks/assets/47995122/28fb7ed6-c6c9-4da8-80f2-ceaf0aff0d6e">

Reply:
Will do, was a md editor making some conflicting changes. should be resolved now.